python/tests/misc.py

```diff
@@ -18,5 +20,91 @@ def test_cmake_files_exists(self):
         )
 
 
+class TestConverter(unittest.TestCase):
+    """
+    Tests the hyperparameter conversions in `rascaline.utils`
+    """
+
+    def test_mode(self):
+        from rascaline.utils import convert_old_hyperparameter_names
+
+        with self.assertRaises(ValueError):
+            convert_old_hyperparameter_names({}, mode="BadMode")
```

Comment:
please also check the error message when checking that exceptions are raised, otherwise it is really easy to get the tests to pass with a different error than the expected one beening raised.
```suggestion
        with self.assertRaises(ValueError) as cm:
            convert_old_hyperparameter_names({}, mode="BadMode")
        
        self.assertEqual(str(cm.exception), "some error message")
```